docs/Course Wiki/ECE Course Offerings/ECE313.md

```diff
@@ -0,0 +1,83 @@
+# ECE313
+
+ECE313 (Probability with Engineering Applications) is a 3-credit-hour course that is specifically required for all ECE majors as part of the Technical Core. [STAT410](../Other%20Course%20Offerings/STAT410.md) may substitute this course. ECE313 is offered in the fall, spring, and summer.
+
+## Content Covered
+
+#### Foundations
+- Probability Spaces
+- Conditional Probability and Independence
+- Counting
+#### Discrete Random Variables
+- Bernoulli, Binomial, Geometric, Poisson, Negative Binomial Distributions
+- Maximum Likelihood Parameter Estimation
+- Binary Hypothesis Testing
+- Confidence Intervals
+- Markov and Chebyshev Inequalities
+#### Continuous Random Variables
+- Uniform, Exponential, Gaussian
+- Functions of Random Variables
+- Failure Rate Functions
+#### Joint Random Variables
+- Correlation, Covariance
+- Functions of Joint Random Variables
+- Sums of Random Variables (convolution)
+- Minimum Mean Square Error Estimation
+- Law of Large Numbers, Central Limit Theorem
+- Joint Gaussian Distributions
+
+The course is roughly split into four sections. The first section, usually regarded as the easiest, deals with the fundamentals of probability (going all the way down to the axioms of probability) and counting. The second sections introduces the idea of a random variable, and starts with discrete random variables. It then covers concepts related to random variables in general, such as mean, variance, independence, conditional probabilities, maximum likelihood, confidence intervals, and binary hypothesis testing. The third part of the course then moves towards continuous random variables, and also covers all of the aforementioned topics, but now through the context of continuous random variables instead of discrete random variables. The final part of the course, infamous for its difficulty, covers joint random variables, along with new topics such as minimum mean square error estimation and the central limit theorem. Each portion of the course builds upon the previous portion.
```

Comment:
"The second sections" -> "The second section"
Get rid of the line phrase "the idea of a"